cmd/release/cmd/config.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"fmt"
+	"html/template"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// configCmd represents the config command
+var configCmd = &cobra.Command{
+	Use:   "config",
+	Short: "",
+	Long:  ``,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			rootCmd.Help()
+			os.Exit(0)
+		}
+
+		switch args[0] {
+		case "generate":
```

Comment:
Using `cmd.AddCommand` adds the command to the help message for the parent command, I think we should use it to avoid having to write it ourselves.
```go
	configCmd.AddCommand(&cobra.Command{
		Use:  "view",
		Long: "View the config",
		Run: func(cmd *cobra.Command, args []string) {
			logrus.Info("test info")
		},
	})
```

```bash
➜  ecm-distro-tools git:(release_command) ✗ ./cmd/release/bin/release-darwin-arm64 config --help
Usage:
  release config [flags]
  release config [command]

Available Commands:
  view        

Flags:
  -h, --help   help for config

Use "release config [command] --help" for more information about a command.
```

Reply:
Does that work though for sub sub commands?

Reply:
Yes, I tested and it worked for `release config view`